packages/react-client/src/ReactFlightClientContext.js

```diff
@@ -27,10 +25,10 @@ export function ServerContextWrapper({ServerContext, value, children}) {
     return ctx;
   }, [parent, value]);
 
-  return (
-    <ServerContextContextProvider value={context}>
-      <ServerContext.Provider value={value}>{children}</ServerContext.Provider>
-    </ServerContextContextProvider>
+  return React.createElement(
```

Comment:
Prefer `React.jsx` since that's now what JSX compiles to. Really people shouldn't do that themselves since we can change the compiler but we are us.